service/frontend/adminHandler.go

```diff
@@ -749,6 +749,10 @@ func (adh *adminHandlerImpl) GetWorkflowExecutionRawHistoryV2(
 		return nil, adh.error(err, scope)
 	}
 	scope = scope.Tagged(metrics.DomainTag(request.GetDomain()))
+	domainName, err := adh.GetDomainCache().GetDomainName(domainID)
```

Comment:
request.GetDomain() is the domain name, you don't need to use domainCache to get the domain name from domainID.